apps/synapse-portal-framework/src/components/PortalContext.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from 'react'
```

Comment:
New file containing types/context that describes a portal configuration.

Pass all of this data into new `Portal` component, and you have a portal.